mods/ra2/rules/allied-naval.yaml

```diff
@@ -76,8 +76,21 @@ dest:
 		LocalOffset: 1152,0,800
 		MuzzleSequence: muzzle-2
 		RequiresCondition: rank-elite
+	Armament@asw:
+		Name: asw
+		Weapon: ASWLauncher
+	CarrierParent:
+		Actors: asw
+		RespawnTicks: 300
+		ArmamentNames: asw
+		LoadedCondition: loaded
+	Exit:
+		SpawnOffset: 768,0,0
 	AttackFrontal:
 		Voice: Attack
+		FacingTolerance: 2
```

Comment:
I don't think this exists anywhere else in this mod. Every attack frontal unit has this in RV to allow firing at moving targets better. I don't think it is a good idea to specifically add this to Destroyer here.